Luke7787/ImmaculateInventors
# finished with adding unique

Summary:
- developed the function to add unique users

Link to any specs used:

How to test:
- run npx nodemon backend.tsx or node backend.tsx
- run mongosh
- type in use inventoryUsers
- open boomerang
- verify the following:
- you can add a user
- add the same user and make sure it gives an error message

Notes: 
format for adding users:
{
    "username": "superman",
    "password": "password123"
}
